src/DynamoCore/BuiltInAndOperators/BuiltIn.xml

```diff
@@ -70,6 +70,7 @@
       <summary>Returns the items from the list that generate the minimum values for the function supplied as the key projector</summary>
       <param name="list">list of values</param>
       <param name="keyProjector">function applied to the list items</param>
+      <returns name="minimumItem">minimum item in list using keyProjector</returns>
```

Comment:
I think we are still deciding on what's a better replacement term for `keyProjector`.

Reply:
Same here: it would be good to include the type, which is `var[]..[]` in the description.

Reply:
> I think we are still deciding on what's a better replacement term for `keyProjector`.

let me know when you decide the name and ill change it.

> Same here: it would be good to include the type, which is `var[]..[]` in the description.

Addressed in 1d53fd1

Reply:
> I think we are still deciding on what's a better replacement term for `keyProjector`.

addressed in 0f91a75